services/document-manager/backend/app/services/transfer_files.py

```diff
@@ -0,0 +1,87 @@
+import numpy
+import uuid
+
+from flask import current_app
+from celery import chord
+
+from app.docman.models.document import Document
+from app.utils.access_decorators import requires_any_of, MINE_ADMIN
+from app.services.object_store_storage_service import ObjectStoreStorageService
+from app.config import Config
+
+from app.utils.tasks import transfer_docs, transfer_docs_result, verify_docs, verify_docs_result
+
+
+def transfer_local_files_to_object_store(wait):
+
+    # Get the documents that aren't stored on the object store (return if they all are)
+    docs = Document.query.filter_by(object_store_path=None).all()
+    if len(docs) == 0:
+        return 'No documents need to be transferred', 200
```

Comment:
Ideally, services shouldn't return 'messages', just booleans or objects. Its up to a controller (the command in this case) to parse that into messages. 

If there are messages that need to be displayed as a result of a non-expected control path they should be Exceptions, which can also be dealt with by the controller if desirable.